production/tmate-common.tf

```diff
@@ -0,0 +1,48 @@
+resource "atlas_artifact" "tmate-edge-gce-image" {
+  name = "travis-ci/tmate-edge"
+  type = "google.image"
+}
+
+resource "atlas_artifact" "tmate-edge-aws-image" {
+  name = "travis-ci/tmate-edge"
+  type = "amazon.image"
+}
+
+resource "google_compute_firewall" "tmate-edge" {
+  name = "tmate-edge-sshd"
+  network = "default"
+
+  allow {
+    protocol = "tcp"
+    ports = ["222"]
+  }
+
+  source_ranges = ["0.0.0.0/0"]
+  target_tags = ["tmate-edge"]
+}
+
+resource "aws_security_group" "tmate-edge" {
+  vpc_id = "vpc-f4188191"
```

Comment:
I think this should be made into a variable, similar to the internal and external zone IDs.